src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

```diff
@@ -0,0 +1,35 @@
+# Scientific research proposal
+
+| Designers     | Implemented | GitHub Links |
+|---------------|-------------|--------------|
+| chromiumboy   | :x: No      | TBD          |
+
+## Problem statement
+At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
+-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 
+-	Due to the high proportion of filler projects at lower research tiers, typically one project will be seen as the ‘optimal’ choice to research (generally either in the industrial or experiment disciplines), leading to many niche items being ignored in the early to mid stages of the shift when they would be most useful, as well as a sense of sameness in technology unlocking despite the research projects being presented at random.
+-	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.
+-	Once all research projects have been completed, there is no longer any compelling reason to conduct any further research.
+
+## Proposed changes
+The changes to scientific research outlined in this proposal have been devised in an attempt to alleviate the issues listed above. These changes include:
+-	Making all low-tier / filler researchable items available to print at round start. These include items such as proximity sensors, TV circuit boards, and surveillance camera equipment.
+-	Making the more desirable mid-tier researchable items (e.g., RCDs, lasers, and high-capacity batteries) also available to print at round start, but with the caveat that science points are required as part of their construction. Lore-wise, this can be explained as the technology already being widely known, however, you need to purchase technology licenses from NT in order to print them. 
```

Comment:
Mentioning RCD here is strategically bad - better omit it. RCD have many very different features, and they lead to many different problems - lets just not get into that swamp of a discussion.